### PR TITLE
Fixed infinite window resizing issue.

### PR DESCRIPTION
**Pull request by ABJECTSELF**
Was unable to properly run the menu because main window would constantly resize every frame by exactly 16 pixels. Discovered that for some reason the setPreferredSize method at line 101 was adding exactly 16 pixels to rectangle1.width every time it was called, resulting in rectangle1 never being equal to rectangle2. Added some padding for comparing the width and height of the two rectangles seemed to resolve the issue for now and should make the menu less laggy. Would like to explore what is causing the discrepancy in the setPreferredSize method later.

### CODE REVIEW

**Comment by IntegratedQuantum** [comment]
I don't think this is the best solution.
Although it solves the problem, other potential problems are introduced:
Changes smaller then the amount of pixels you chose will not be registered correctly which could result in  a small portion of the frame being empty or a part from the game getting cut away on the border, which looks ugly.

I'm not entirely sure, but I think the problem is that the frame size also includes the decoration size(the borders on all 4 sides) in width/height measurement whereas the panel is only the region inside the borders.(Maybe simply using main.getBounds instead of frame.getBounds might resolve the issue).

If the above doesn't help I think it would be best to store the last measured rectangle instead of the preferred one for comparison.

File: src/entertheblack/menu/Main.java [hunk 1]
```diff
@@ -87,15 +87,19 @@ public static void main(String[] args) throws InterruptedException {
...
 			rectangle1 = frame.getBounds();
-			if (rectangle1 != rectangle2) {
+			if (Math.abs(rectangle1.width-rectangle2.width)>widthLimiter
+			|| Math.abs(rectangle1.height-rectangle2.height)>heightLimiter) {
 				main.curWidth = rectangle1.width;
 				main.determineOptimalScreenSize();
 				rectangle1.height = main.optimalHeight;
 				rectangle2 = rectangle1;
 				main.setPreferredSize(new Dimension(rectangle1.width, rectangle1.height));
+				//System.out.println(Integer.toString(rectangle1.width)); Uncomment to track changes in window size.
```
Comment (IntegratedQuantum) [inline src/entertheblack/menu/Main.java:102]: A small note on keeping debug code as comments:
This code usually will never get used again and if it is used again, it wouldn't be too hard to rewrite, but on the other hand keeping junk code everywhere decreases readability.

**Comment by IntegratedQuantum** [comment]
Fixed in 9cab7af.
The problem in fact was that I used the size of the frame to resized the size of the panel which is wrong.
Thanks for pointing out this issue.

**Comment by ABJECTSELF** [comment]
Glad I could help. I'm still practicing with creating GUIs in Java so this
was a good tip to learn.

**Pull request closed without merging**